Gemfile

```diff
@@ -311,6 +311,11 @@ gem 'roar', '~> 1.2.0'
 # CORS for API
 gem 'rack-cors', '~> 1.1.1'
 
+# Gmail API
+gem 'google-apis-core', require: false
```

Comment:
This line should be removed with `require: false`, otherwise bundler will complain as `google-apis-gmail_v1` will already require the gem (that's why the tests are failing)

Reply:
@Givo29 this is still causing the tests to fail, simply remove that first require should allow the suite to pass.

Reply:
Ah sorry, I didn't see that comment. I will fix it now.